projects/aws/bottlerocket-bootstrap/pkg/files/templater.go

```diff
@@ -0,0 +1,32 @@
+package files
+
+import (
+	"bytes"
+	"io/fs"
+	"text/template"
+
+	"github.com/pkg/errors"
+)
+
+func ExecuteTemplate(content string, data interface{}) ([]byte, error) {
+	temp := template.New("tmpl")
+	temp, err := temp.Parse(content)
+	if err != nil {
+		return nil, errors.Wrap(err, "Error parsing template")
```

Comment:
Can we remove all mentions of "Error" through this diff? I think we wanted to enforce that now

Reply:
I'd like to do that in a different PR, or this PR will change every single file